bedrock/firefox/templates/firefox/new/fx-lifestyle/base.html

```diff
@@ -0,0 +1,83 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+ # License, v. 2.0. If a copy of the MPL was not distributed with this
+ # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
+
+{% extends "firefox/base-resp.html" %}
+
+{% block page_title_prefix %}{% endblock %}
+{% block page_title %}{{_('Free Web Browser for Android, iOS, Desktop | Firefox')}}{% endblock %}
+{% block page_desc %}{{_('Millions of people around the world trust Firefox Web browsers on Android, iOS and desktop computers. Fast. Private. Download now!')}}{% endblock %}
+
+{% block page_css %}{% endblock %}
+
+{% block body_id %}firefox{% endblock %}
+{% block body_class %}{% endblock %}
+
+{% block site_header %}{% endblock %}
+
+{% block content %}
+
+<section class="content-section" id="intro">
+  <div class="content">
+    <div class="copy">
+      <img src="{{ static('img/firefox/new/firefox-logo.png') }}" />
+      {% block head_content %}
+      {% endblock %}
+
+      <div id="download-button-wrapper-scene1">
+        {{ download_firefox( locale_in_transition=true, dom_id="download-firefox") }}
+      </div>
+
+      {# this button is used only for scene 2's #}
+      <div class="download-button-wrapper desktop" id="download-button-wrapper-desktop">
+        {{ download_firefox(force_direct=true, locale_in_transition=true, dom_id='primary-download-button') }}
```

Comment:
Hmm, this `locale_in_transition` shouldn't be needed for direct download links, right?